src/test/regress/expected/window_functions_0.out

```diff
@@ -0,0 +1,1591 @@
+-- ===================================================================
```

Comment:
Is this PG version difference? The main difference seems PG 13 has incremental sort:
```
1475c1478
<                            ->  Sort
---
>                            ->  Incremental Sort
1476a1480
>                                  Presorted Key: users_table.user_id
1483c1487
< (17 rows)
---
> (18 rows)
```

Reply:
yes